### DISCUSSION TITLE
Accessing public-facing s3 bucket?

### DISCUSSION THREAD

**Discussion by DruidSmith** [aws/aws-sdk-pandas#1740] [General]
Awswrangler appears to handle credentials via [session](https://aws-sdk-pandas.readthedocs.io/en/stable/tutorials/002%20-%20Sessions.html) , but I am trying to access data in some public facing open data buckets that I do not own and do not have credentials for.  How can that scenario be best handled?

I've tried things like setting boto3 session with aws_access_key_id=None, aws_secret_access_key=None and that doesn't work, it still complains about not having credentials.  I was trying to see if I could use signature_version=UNSIGNED from botocore UNSIGNED but that appears to only apply to boto3.client and not boto3 session.

Any tips/example (and potential update to the documentation) would be greatly appreciated.

**Comment by malachi-constant** [comment]
I'm fairly certain there is no way to establish a boto3 session without credentials. In the case of a public s3 bucket, any aws credentials with permissions to access S3 will work for you.
